docs/pages/services/onto-docker.md

```diff
@@ -34,27 +33,27 @@ git clone https://git.material-digital.de/apps/ontodocker.git
 cd ontodocker
 ```
 
-### 2. Copy compose template
+### 2. Connect to SSO Identity Provider (IDP)
 
-```bash
-# copy (or link) the compose file template
-cp docker-compose-prod.yml docker-compose.yml
-```
+If you alreday have a valid client_secrets.json already, you must copy it to ```./data/oidc/```
+
+Otherwise in order to connect OntoDocker to the IDP you might need an initial access token (IAT) or generate one, if you want to connect it to your local Instance. (see "Initial Access Token" section of the [Keycloak manual](https://www.keycloak.org/docs/latest/securing_apps/#_initial_access_token))
```

Comment:
```suggestion
In order to connect OntoDocker to the IDP you might need an initial access token (IAT), which you will receive from the maintainer of the IDP,  or generate one, if you want to connect it to your local Instance. (see "Initial Access Token" section of the [Keycloak manual](https://www.keycloak.org/docs/latest/securing_apps/#_initial_access_token))
```

we should probably add a section about IAT's in our deployment guide but for now Keycloak manual is also fine